components/esp_insights/src/esp_insights.c

```diff
@@ -91,6 +91,7 @@ typedef struct {
 
 static const char *TAG = "esp_insights";
 static esp_insights_data_t s_insights_data;
+static esp_insights_entry_t *periodic_insights_entry;
```

Comment:
Please prefix the static variable name with `s_`
```suggestion
static esp_insights_entry_t *s_periodic_insights_entry;
```